### DISCUSSION TITLE
DrawTextureTiled will be removed from latest raylib version ?

### DISCUSSION THREAD

**Discussion by popolony2k** [raysan5/raylib#2818] [Q&A] [answered]
Hi all...I'm using raylib as backend to my top level game engine and I already had made some implementations using DrawTextureTiled, handling all it's behaviors to fit my engine.

Last month was the latest time I compiled my project and today I'm trying to compile it again and it's is complaining about missing of DrawTextureTiled function (since my ibrary goes to raylib repository and dowload the master repository to compile).

So, my question is. Is DrawTextureTiled deprecated and is really removed ? Which is the new function that I can use with the exact same behavior ?

Regards

**Comment by raysan5** [comment, marked as answer]
@popolony2k that function has been moved to example `textures_draw_tiled`, you can just copy it from there into your code.

**Reply by popolony2k** [reply]
@raysan5 Thanks a lot. I'll do that.

**Comment by orcmid** [comment]
**Side comment:** Because breaking changes can occur on the development branch, I would recommend always building with stable releases of `raylib`.  This also gives you a fallback if building with a new release confronts you with breaking changes or even new bugs.  You then have time to work out an upgrade to the latest release or simply wait for the next latest `raylib` release.  Unless you want/need to work on `raylib` features yourself, I suggest staying clear of the developer branch.  (PS: It would also be useful if the `raylib` project used semantic versioning so you'd know when to expect breaking changes and find them in the release notes.)

**Reply by popolony2k** [reply]
@orcmid Thanks a lot. I'm using master because my development still doesn't have a stable version so I would like to test the fresh features of raylib main branch :)
But you're 100% right, the safest way to integrate anything is using a stable version.

Regards
